File: Sim06-EsameInClasse/src/Auto.java
public class Auto extends Veicolo {
    double capienza_bagagliaio;
    int cilindrata;
    String categoria;




        
    public Auto(int codice, String tipo, String targa, String modello, String marca, double capienza_bagagliaio,
            int cilindrata, String categoria) {
        super(codice, tipo, targa, modello, marca);
        this.capienza_bagagliaio = capienza_bagagliaio;
        this.cilindrata = cilindrata;
        this.categoria = categoria;
    }

    public double getCapienza_bagagliaio() {
        return capienza_bagagliaio;
    }

    public int getCilindrata() {
        return cilindrata;
    }

    public String getCategoria() {
        return categoria;
    }
//"tipo, targa, codice, modello, marca, costo giornaliero,bagagliaio, cilindrata, categoria, rimorchio, numero di posti,vano di carico"
    public String toString(){
        return "Automobile "+targa+codice+modello+marca+costo_giornaliero+capienza_bagagliaio+categoria+" - "+" - "+" - ";
    }
}

File: 40/Auto.py
from Veicoli import Veicoli

class Auto(Veicoli):
    def __init__(self, codice, tipo, targa, modello, marca, costoGiornaliero,cilindrata,capienza):
        super().__init__(codice, tipo, targa, modello, marca, costoGiornaliero)
        self.cilindrata = cilindrata
        self.capienza =capienza

    "tipo, targa, codice, modello, marca, costo giornaliero,cilindrata, bagagliaio, categoria, numero di posti, vano di carico"
    def __str__(self) -> str:
        return self.tipo+"\t"+self.targa+"\t"+str(self.codice)+"\t"+self.modello+"\t"+self.marca+"\t"+str(self.costoGiornaliero)+"\t"+str(self.cilindrata)+"\t"+str(self.capienza)+"\t-"+"\t-"+"\t-"
    